### ISSUE TITLE
`Cmd + B` does not run scene while in script editor on Mac OS

### ISSUE THREAD

**Issue by tavurth** [godotengine/godot#56961]
### Godot version

v4.0.dev.20220118.official [bc5ba38ff]

### System information

Mac OSX (Intel 2019)

### Issue description

Normally in 3.4.2 you can use `Cmd + B` to run the project, but this does not seem to function properly in the script editor on 4.x even when the editor window has focus.

### Steps to reproduce

1. Enter script editor
2. Press `Cmd + B`

**Comment by Calinou** [comment]
cc @Paulb23

**Comment by nabil-alsabah** [comment]
This issue hasn't been resolved as of Beta 2.

**Comment by konsuko** [comment]
I also noticed that other keyboard shortcuts don't seem to be working when inside the editor.
E.g. `Cmd + ,` (to open the Editor Settings) does also not work when in the built-in editor.

**Comment by nabil-alsabah** [comment]
I found a way to solve the problem: The problem occurs because the same shortcut is linked to several actions. Thus, go to editor settings -> shortcuts to resolve the issue by assigning one short cut to one action.

**Comment by konsuko** [comment]
> I found a way to solve the problem: The problem occurs because the same shortcut is linked to several actions. Thus, go to editor settings -> shortcuts to resolve the issue by assigning one short cut to one action.

Holy ****, thanks! That's been bugging me for soo long 👯

**Comment by EricEzaM** [comment]
This is a clash with `ScriptTextEditor/Go To Next Bookmark`, right? Can we just add an override for that shortcut on MacOS to be something else>

**Comment by lostminds** [comment]
> This is a clash with ScriptTextEditor/Go To Next Bookmark, right? 

This indeed seems to be the case. I tried it now, and setting the Go to next bookmark shortcut to the same key as Run project shortcut prevents me from running the project when the code editor is active.

But shouldn't the clash happen on all platforms? As on both shortcuts would be Ctrl-B by default?
Or maybe this indicates that shortcut key events are handled differently or in different order based on platform?

Either way, it sounds like a good idea to change the default shortcut for Go to next bookmark, and it should fix this issue for now.

**Comment by thloh85** [comment]
> 

Not really. On windows, as what I remember, the "run project" command is actually F5, not Ctrl+B. 
Personally I don't like MacOS's keybinding (cmd+c instead of ctrl+c).

**Comment by Alex2782** [comment]
I have checked the version 3.5.2. (MacOS)
There, `Play` and `Go to Next Bookmark` are triggered simultaneously.  

Should it work like this again? 
Or better assign the "Default Shortcut"  for `Go to Next Bookmark` e.g. with `Ctrl + Command + B`? 
To avoid conflict with `Play`.


In the video I first executed `Toogle Bookmark` (blue dots)

https://github.com/godotengine/godot/assets/41921395/e117b2bb-2043-4734-9356-08e0e68d13ed


**Default Shortcuts (MacOS)**
![image](https://github.com/godotengine/godot/assets/41921395/0de8010a-c8a3-437b-9e29-3d047dca9786)

![image](https://github.com/godotengine/godot/assets/41921395/3feb1e05-e4fb-4ace-ae5b-82fbde6f4af5)

**Comment by ShlomiRex** [comment]
v4.1
The Cmd+B can't be assigned to play since 'script_text_editor/goto_next_bookmark' is triggered.
